packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -918,18 +929,18 @@ export async function cleanup(compass?: Compass): Promise<void> {
   }
 
   let timeoutId;
-  const timeoutPromise = new Promise<void>((resolve) => {
+  const timeoutPromise = new Promise<'timeout'>((resolve) => {
     timeoutId = setTimeout(() => {
-      console.error('It took too long to close compass');
-      resolve();
+      console.error(`It took too long to close compass [${compass.name}]`);
+      resolve('timeout');
     }, 30000);
   });
 
-  const closePromise = (async function close(): Promise<void> {
+  const closePromise = (async function close(): Promise<'closed'> {
     try {
       await compass.stop();
     } catch (err) {
-      debug('An error occurred while stopping compass:');
+      debug(`An error occurred while stopping compass: [${compass.name}]`);
```

Comment:
Without these debug tags, it's quite a bit harder to identify _which_ Compass process didn't exit on its own